packages/react-magma-dom/src/components/InputBase/index.tsx

```diff
@@ -387,6 +431,28 @@ export const InputBase = React.forwardRef<HTMLInputElement, InputBaseProps>(
             />
           </IconButtonContainer>
         )}
+
+        {isClearable && value && (
+          <IsClearableContainer theme={theme}>
```

Comment:
When I tab through the Search field, I notice the Search button gets focused before the X button, which doesn't match what I'm seeing visually.  Can we switch the order of these elements in the dom so it matches the visual?

Reply:
Awesome, this looks great!  

One other thing I hadn't noticed before is that after the clicking the 'clear' icon, nothing is in focus. I think we should return the focus to the search box.  

This PR also needs a changeset.  Everything else looks good to me.